### PR TITLE
Add mime-type `image/avif` and `image/avifs`

### PR DESCRIPTION
**Pull request by gsouquet**
re: #220 

Looking at the page on [CORS enabled images](https://wiki.developer.mozilla.org/en-US/docs/Web/HTML/CORS_enabled_image) on MDN I realised that h5bp did not mention AVIF images at all. I have seen that an issue was opened and decided to open a PR to add both mime types

* `.avif`  => `image/avif`
* `.avifs` => `image/avif-sequence`

I followed the same default as the WEBP format, and I believe this covers most of it

Keen to hear your feedback on this

### CODE REVIEW

File: dist/.htaccess [hunk 1]
```diff
@@ -43,7 +43,7 @@
```
Comment (LeoColomb) [inline dist/.htaccess:43]: Please don't edit `.htaccess`

File: h5bp/cross-origin/images.conf [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 <IfModule mod_setenvif.c>
     <IfModule mod_headers.c>
-        <FilesMatch "\.(bmp|cur|gif|ico|jpe?g|a?png|svgz?|webp)$">
+        <FilesMatch "\.(bmp|cur|gif|ico|jpe?g|a?png|svgz?|webp|avifs?)$">
```
Comment (LeoColomb) [inline h5bp/cross-origin/images.conf:12]: 👍 but better to keep alphabetic ordering, so probably at the first place in the array

```suggestion
        <FilesMatch "\.(avifs?|bmp|cur|gif|ico|jpe?g|a?png|svgz?|webp)$">
```

File: h5bp/web_performance/filename-based_cache_busting.conf [hunk 3]
```diff
@@ -12,5 +12,5 @@
 <IfModule mod_rewrite.c>
     RewriteEngine On
     RewriteCond %{REQUEST_FILENAME} !-f
-    RewriteRule ^(.+)\.(\w+)\.(bmp|css|cur|gif|ico|jpe?g|m?js|a?png|svgz?|webp|webmanifest)$ $1.$3 [L]
+    RewriteRule ^(.+)\.(\w+)\.(bmp|css|cur|gif|ico|jpe?g|m?js|a?png|svgz?|webp|avifs?|webmanifest)$ $1.$3 [L]
```
Comment (LeoColomb) [inline h5bp/web_performance/filename-based_cache_busting.conf:15]: 👍 but better to keep alphabetic ordering, so probably at the first place in the array

```suggestion
    RewriteRule ^(.+)\.(\w+)\.(avifs?|bmp|css|cur|gif|ico|jpe?g|m?js|a?png|svgz?|webp|webmanifest)$ $1.$3 [L]
```

File: h5bp/media_types/media_types.conf [hunk 4]
```diff
@@ -42,6 +42,8 @@
     AddType image/bmp                                   bmp
     AddType image/svg+xml                               svg svgz
     AddType image/webp                                  webp
+    AddType image/avif                                  avif
+    AddType image/avif-sequence                         avifs
```
Comment (LeoColomb) [inline h5bp/media_types/media_types.conf:46]: 👍 but better to keep alphabetic ordering, so probably as first image type

File: h5bp/web_performance/cache_expiration.conf [hunk 5]
```diff
@@ -75,6 +75,8 @@
     ExpiresByType image/png                             "access plus 1 month"
     ExpiresByType image/svg+xml                         "access plus 1 month"
     ExpiresByType image/webp                            "access plus 1 month"
+    ExpiresByType image/avif                            "access plus 1 month"
+    ExpiresByType image/avif-sequence                   "access plus 1 month"
```
Comment (LeoColomb) [inline h5bp/web_performance/cache_expiration.conf:79]: 👍 but better to keep alphabetic ordering, so probably as first image type

**Review by LeoColomb — CHANGES_REQUESTED** [review]
Thanks a lot for this PR @gsouquet 

Please see my few remarks.

**Comment by gsouquet** [comment]
Thank you for the quick review @LeoColomb 👍 

I hadn't notice the alphabetical ordering. This is now fixed
Also reverted `dist/.htaccess` as it seems that it is handled by `.github/workflows/release.yml`

**Review by LeoColomb — APPROVED** [review]
Thanks again, much better 😃

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**